runtime/src/main/scala/com/ing/baker/runtime/core/interations/MethodInteractionImplementation.scala

```diff
@@ -76,7 +76,7 @@ object MethodInteractionImplementation {
 
     // when the interaction does not fire an event, Void or Unit is a valid output type
     if (interaction.eventsToFire.isEmpty && output == null)
-      RuntimeEvent(interaction.interactionName, Seq.empty)
+      null
```

Comment:
Dont we have a better return type for this?

Reply:
for now we use null for this.
An issue is created to remove null use and instead use an Option.